weather.py

```diff
@@ -46,8 +55,11 @@ def calculate_mean(weather_data):
     Returns:
         A float representing the mean value.
     """
-    pass
-
+    #map() loops through data & runs a function on each item , syntax: map(function, iterables)
+    newList = (map(float, weather_data))
+    #to use .mean import statistics package
+    mean = statistics.mean(newList)
```

Comment:
- Nice use of importing new modules and applying `map()`.
- Generally in Python we use `lower_case_with_underscores` for variable naming to improve readability (`mixedCase` isn't seen as often). i.e. `new_list` instead of `newList`. Check out [PEP8](https://peps.python.org/pep-0008/) if you're interested in learning more.